vshard/storage/init.lua

```diff
@@ -3721,6 +3736,7 @@ return {
     cfg = function(cfg, uuid) return storage_cfg_fiber_safe(cfg, uuid, false) end,
     on_master_enable = storage_make_api(on_master_enable),
     on_master_disable = storage_make_api(on_master_disable),
+    on_bucket_event = storage_make_api(on_bucket_event),
```

Comment:
This one probably should be allowed to call even before `vshard.storage.cfg` starts and ends. Otherwise rebalancing might start between `cfg` ends and when a user manages to set the triggers.

Reply:
Actually the same thing applies to `on_master_enable/disable`. AFAICS, by it is supposed to be executed before `cfg` by design. I think we should remove `storage_make_api` for them to.